src/integrationTest/java/uk/gov/hmcts/darts/cases/service/RetentionPolicyTest.java

```diff
@@ -0,0 +1,180 @@
+        assertThat(singleCase.getRetentionDateTimeApplied()).isEqualTo(caseRetention.getRetainUntilAppliedOn());
+    }
+
+    @Test
+    void threePoliciesTwoWithCurrentStateCompleteAndOnePending() {
+        var minimalHearing = createSomeMinimalHearing();
+        List<CaseRetentionEntity> caseRetentionList = new ArrayList<>();
+        caseRetentionList.add(createCompleteCaseRetention1(minimalHearing.getCourtCase()));
+        caseRetentionList.add(createCompleteCaseRetention2(minimalHearing.getCourtCase()));
+        caseRetentionList.add(createPendingCaseRetention(minimalHearing.getCourtCase()));
+        minimalHearing.getCourtCase().setCaseRetentionEntities(caseRetentionList);
+
+        var hearingEntity = dartsDatabase.saveRetentionsForHearing(minimalHearing, caseRetentionList);
+        var singleCase = casesMapper.mapToSingleCase(hearingEntity.getCourtCase());
+
+        CaseRetentionEntity caseRetention = createCompleteCaseRetention2(hearingEntity.getCourtCase());
+        assertThat(singleCase.getRetentionPolicyApplied()).isEqualTo(caseRetention.getRetentionPolicyType().getPolicyName());
+        assertThat(singleCase.getRetainUntilDateTime()).isEqualTo(caseRetention.getRetainUntil());
+        assertThat(singleCase.getRetentionDateTimeApplied()).isEqualTo(caseRetention.getRetainUntilAppliedOn());
+    }
+
+    @Test
+    void noRentionPolicyAdded() {
+        var minimalHearing = createSomeMinimalHearing();
+        List<CaseRetentionEntity> caseRetentionList = new ArrayList<>();
+        var hearingEntity = dartsDatabase.saveRetentionsForHearing(minimalHearing, caseRetentionList);
+        var singleCase = casesMapper.mapToSingleCase(hearingEntity.getCourtCase());
+
+        assertThat(singleCase.getRetentionPolicyApplied()).isNull();
+        assertThat(singleCase.getRetainUntilDateTime()).isNull();
+        assertThat(singleCase.getRetentionDateTimeApplied()).isNull();
+    }
+
+    @Test
+    void onePolicyWithCurrentStatePending() {
+        var minimalHearing = createSomeMinimalHearing();
+        minimalHearing.getCourtCase().setCaseClosedTimestamp(OffsetDateTime.now());
+        List<CaseRetentionEntity> caseRetentionList = new ArrayList<>();
+        caseRetentionList.add(createPendingCaseRetention(minimalHearing.getCourtCase()));
+        minimalHearing.getCourtCase().setCaseRetentionEntities(caseRetentionList);
+
+        var hearingEntity = dartsDatabase.saveRetentionsForHearing(minimalHearing, caseRetentionList);
+        var singleCase = casesMapper.mapToSingleCase(hearingEntity.getCourtCase());
+
+        assertThat(singleCase.getRetentionPolicyApplied()).isNull();
+        assertThat(singleCase.getRetainUntilDateTime()).isNull();
+        assertThat(singleCase.getRetentionDateTimeApplied()).isNull();
+        assertThat(singleCase.getCaseClosedDateTime()).isEqualTo(hearingEntity.getCourtCase().getCaseClosedTimestamp());
+    }
+
+    @Test
+    void retrievePolicyDetails() {
```

Comment:
Consider moving this to a unit test for the Policy enum as it isn't an integration test

Reply:
Agreed, moved to unit test